src/main/java/frc/robot/subsystems/ArmTelescopicSystem.java

```diff
@@ -0,0 +1,85 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.spark.SparkBase;
+import com.revrobotics.spark.SparkClosedLoopController;
+import com.revrobotics.spark.SparkLowLevel;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.SparkBaseConfig;
+import com.revrobotics.spark.config.SparkMaxConfig;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class ArmTelescopicSystem extends SubsystemBase {
+    private final SparkMax motor;
+    private final RelativeEncoder encoder;
+    private final SparkClosedLoopController pid;
+
+    public ArmTelescopicSystem() {
+        motor = new SparkMax(RobotMap.ARM_TELESCOPIC_MOTOR_ID, SparkLowLevel.MotorType.kBrushless);
+        encoder = motor.getAlternateEncoder();
```

Comment:
```suggestion
        encoder = motor.getEncoder();
```
We're using the NEO-integrated encoder, not an alternate encoder.

Reply:
done